app/components/CollapsibleDisplayContent/CollapsibleDisplayContent.css

```diff
@@ -0,0 +1,43 @@
+@import url('~app/styles/variables.css');
+
+.collapse {
+  position: relative;
+  overflow: hidden;
+  transition: 0.5s var(--easing) height;
```

Comment:
Use `transition: height var(--easing-slow) instead`, please. It's important to not use different speeds everywhere.